scripts/test/launchTestUtilities.ts

```diff
@@ -114,67 +196,11 @@ export async function getCliArgsToDisableExtensions(
     return ids
 }
 
-export function getMinVsCodeVersion(): string {
+function getMinVsCodeVersion(): string {
     const vsCodeVersion = packageJson.engines.vscode
 
     // We assume that we specify a minium, so it matches ^<number>, so remove ^'s
     const sanitizedVersion = vsCodeVersion.replace('^', '')
     console.log(`Using minimum VSCode version specified in package.json: ${sanitizedVersion}`)
```

Comment:
This log message is the only thing that differs from  `shared/vscode/env.ts:getMinVscodeVersion`. So this function can re-use that one, and only this log message would be added here.